pruner/example_programs/fabric_with_creduce/fabric.p4.orig

```diff
@@ -0,0 +1,169 @@
+/*
```

Comment:
Do we need to duplicate all of this? It is the same code as in the other folder right? We can just share it.

Reply:
we don't need to duplicate the include folder, but creduce modifies the original file and moves it to `file_name.orig`, so thatswhy I isolate both of them.

Reply:
That is unfortunate, I just checked and there does not seem to be an option to flip this. One thing we could do is to have a creduce script that makes a temporary copy that is reduced and use the --tidy flag.

Reply:
Alright